### ISSUE TITLE
CPU usage is getting to 90% at the time of backup

### ISSUE THREAD

**Issue by raftaar1191** [Automattic/jetpack#20124]
![image](https://user-images.githubusercontent.com/22215595/81039644-4118c580-8ec7-11ea-8932-6b237589924d.png)

If we are using any cloud fair server and if we did not get a response from the server within 100 section then it thought and error and that error is coming

**Comment by jeherve** [comment]
@rogercoathup Could you contact our support team to give us more information about your site? You should see a contact form in your VaultPress dashboard:
https://dashboard.vaultpress.com/contact/

Thank you.

**Comment by hubertnguyen** [comment]
Hi @jeherve , I noticed that Vaultpress triggers a lots of calls to this:

`/wp-admin/?action=uploads%3Als&doing_wp_cron=&pe=1&vaultpress=true&vector=1627356854.1834&wp-admin`

It seems that it has to do with the uploads items, and I have hundreds of thousands. Similar calls happen at a rate of about 15 per minute. 

Since I have already provided sFTP and MySQL direct access, I was wondering what these calls were for? It's likely what some people see as high-CPU events
